stingray/stats.py

```diff
@@ -0,0 +1,268 @@
+import warnings
+from collections.abc import Iterable
+
+import numpy as np
+from scipy import stats
+
+
+__all__ = ['p_multitrial_from_single_trial',
+           'p_single_trial_from_p_multitrial',
+           'fold_profile_probability',
+           'fold_detection_level',
+           'z2_n_detection_level',
+           'z2_n_probability']
+
+
+def p_multitrial_from_single_trial(p1, n):
+    """Calculate a multi-trial probability from a single-trial one.
+
+    Calling _p_ the probability of a single hit, the Binomial
+    distributions says that the probability _at least_ one outcome
+    in n trials is
+
+                         n
+    P (k ≥ 1) =   Σ    (   ) p^k (1 - p)^(n - k)
+                k ≥ 1    k
+
+    or more simply, using P(k ≥ 0) = 1
+                          n
+    P (k ≥ 1) =   1 -   (   ) (1 - p)^n = 1 - (1 - p)^n
+                          0
+    """
+    return 1 - (1 - np.longdouble(p1))**np.longdouble(n)
+
+
+def p_single_trial_from_p_multitrial(pn, n):
```

Comment:
Needs precise definition in the docstring of what `pn` and `n` are.

Reply:
Done